v2/internal/runner/options.go

```diff
@@ -195,9 +186,65 @@ func validateOptions(options *types.Options) error {
 		return errors.New("ipv4 and/or ipv6 must be selected")
 	}
 
+	// Validate cloud option
+	if err := validateCloudOptions(options); err != nil {
+		return err
+	}
 	return nil
 }
 
+func validateCloudOptions(options *types.Options) error {
+	if options.ScanList || options.DeleteScan != "" || options.ScanOutput != "" || options.ListDatasources || options.ListTargets || options.ListTemplates || options.RemoveDatasource != "" || options.AddTarget != "" || options.AddTemplate != "" || options.RemoveTarget != "" || options.RemoveTemplate != "" || options.GetTarget != "" || options.GetTemplate != "" {
+		if !options.Cloud {
```

Comment:
This needs to be specified explicity by the user as discussed with @ehsandeep

Reply:
@Mzack9999 Usually, we prefer to enable things internally when dependent options are used, but In this case, we wanted to keep a clear separation between local and cloud run to avoid confusion or mistakenly mixing up to trigger cloud runs.

for simple flow, it's possible for users to do `export NUCLEI_CLOUD=true` and run cloud scan all the time without adding cloud option all the time.